container/rkt/client.go

```diff
@@ -37,6 +38,13 @@ var (
 
 func Client() (rktapi.PublicAPIClient, error) {
 	once.Do(func() {
+		_, err := net.Dial("tcp", "localhost:15441")
+		if err != nil {
+			rktClient = nil
+			rktClientErr = fmt.Errorf("rkt: cannot connect to rkt api service: %v", err)
```

Comment:
Use a different error message here so the failures can be distinguished. Maybe "Unable to dial rkt api at localhost:15441"